### ISSUE TITLE
missing get_install_tpls_args function

### ISSUE THREAD

**Issue by rfiorella** [ecoon/ats_manager#1]
Hey @ecoon - just a quick note that bin/install_tpls.py on the tpls separately branch fails as there's no get_install_tpls_args function in ui.py. Maybe this is in a commit that hasn't been pushed yet?

**Comment by ecoon** [comment]
Yes, clearly install_tpls hasn't been updated.  I'll update it.  Also need to merge this branch to master, it's definitely the stable branch now (not master).
